frontend/components/Notebook.js

```diff
@@ -66,6 +64,7 @@ let CellMemo = ({
         // Object references may invalidate this faster than the optimal. To avoid this, spread out objects to primitives!
         cell_id,
         metadata.disabled,
+        metadata.show_logs,
```

Comment:
@disberd not sure if you already know, but i think this where the memo is and the `skip_as_script` property should be added to re-render a cell.

Reply:
Thanks!
Indeed yesterday @pankgeorg showed me this part :)